sway/input/seatop_down.c

```diff
@@ -107,6 +107,8 @@ void seatop_begin_down(struct sway_seat *seat, struct sway_container *con,
 	seat->seatop_impl = &seatop_impl;
 	seat->seatop_data = e;
 
-	container_raise_floating(con);
+	if (con) {
+		container_raise_floating(con);
+	}
 	transaction_commit_dirty();
```

Comment:
I don't think we need to commit the transaction if we haven't raised the container. We should also leave a comment here reminding us that `con` is null when clicking a layer surface.